sdk/loadtestservice/Azure.Developer.LoadTesting/samples/Sample1_CreateOrUpdateTestAsync.md

```diff
@@ -2,21 +2,22 @@
 
 To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/loadtestservice/Azure.Developer.LoadTesting/README.md#getting-started) for details.
 
-You can create a LoadTestclient and call the `CreateOrUpdateTest` method from client `LoadTestAdministrationClient`
+The sample below demonstrates how to create a new load test using the `LoadTestAdministrationClient` client.
 
 ## Create LoadTestAdministrationClient
 ```C# Snippet:Azure_Developer_LoadTesting_CreateAdminClient
-string endpoint = TestEnvironment.Endpoint;
-Uri enpointUrl = new Uri("https://"+endpoint);
-TokenCredential credential = TestEnvironment.Credential;
+// The data-plane endpoint is obtained from Control Plane APIs with "https://"
+Uri endpointUrl = new Uri("https://data-plane-guid.region.cnt-prod.loadtesting.azure.com");
+TokenCredential credential = new DefaultAzureCredential();
 
 // creating LoadTesting Administration Client
-LoadTestAdministrationClient loadTestAdministrationClient = new LoadTestAdministrationClient(enpointUrl, credential);
+LoadTestAdministrationClient loadTestAdministrationClient = new LoadTestAdministrationClient(endpointUrl, credential);
 ```
 
 ## Calling CreateOrUpdateTest
 ```C# Snippet:Azure_Developer_LoadTesting_CreateOrUpdateTestAsync
 string testId = "my-test-id";
+Uri keyVaultSecretUrl = new Uri("https://sdk-testing-keyvault.vault.azure.net/secrets/sdk-secret");
```

Comment:
Maybe we should demonstrate how to get the URI once, and then just have the other samples refer to the endpointUri variable going forward.

Reply:
Sure @christothes, however this Uri could be any keyVaultSecretUri